src/utils/diagnostic.ts

```diff
@@ -0,0 +1,63 @@
+import { Diagnostic, DiagnosticSeverity, Range } from 'vscode-languageserver';
+import { TemplateLinterError } from '../template-linter';
+
+const ParseErrorExp = /^Parse error on line (\d+)/;
+const OnLineErrorExp = / \(on line (\d+)\)\.$/;
+
+export function toDiagnostic(source: string, error: TemplateLinterError): Diagnostic {
+  return {
+    severity: DiagnosticSeverity.Error,
+    range: toRange(source, error),
+    message: toMessage(error),
+    source: error.rule ? 'ember-template-lint' : 'glimmer-engine'
```

Comment:
@t-sauer `glimmer-engine` could also be a `handlebars` how far down the rabbit whole do you want to go with this 😄